unskript-ctl/unskript-client.py

```diff
@@ -363,20 +363,22 @@ def run_ipynb(filename: str, status_list_of_dict: list = None, filter: str = Non
     status_list_of_dict.append(status_dict)
 
 
-def run_checks(filter: str):
+def run_checks(args: list):
     """run_checks This function takes the filter as an argument
     and based on the filter, this function queries the DB and
     creates a temporary runnable runbooks and run them, updates
     the audit results.
 
-    :type filter: string
-    :param filter: Filter string, possible values are all,<connector>,failed
+    :type args: list
+    :param args: input list, possible values are all,<connector>,failed or -f <check_name>
```

Comment:
the cli using a mix of arguments without - and with - doesnt sound. Can we have
```
unskript-ctl -rc 
     -a (--all) 
     -t (--type) <allowed values, look at add_creds.py> 
     --failed 
     -c(--check) <check name>
```

Reply:
failed should be boolean, default to false.

Reply:
Done